app/views/data_exports/_new_analysis_export_dialog.html.erb

```diff
@@ -58,7 +58,7 @@
           </div>
```

Comment:
The `Workflow Executions (NUM selected)` button class `rounded-t` in this file needs to be updated to `rounded-t-lg`.

Reply:
Thanks, updated in f4e5af291db807334c9a2e5f665fa306dc85cba9